Bitwise_Technology_Gabriel_Oliveira.py

```diff
@@ -0,0 +1,71 @@
+import Consultar_Informações_Cadastro
+import requests
+import Verificar_usuarios_entidades
+import Banco_de_dados
+import os
+
+# Criando o banco de dados
+create_db = not os.path.isfile('Bitwise_Gabriel_Oliveira.db')
+if create_db:
```

Comment:
Seria mais indicado ter adicionado aquele not aqui no if e alterado o nome dela (variável) para `_db`.